### PR TITLE
updating cve advisory for kops-1.7.1 release

### PR DESCRIPTION
**Pull request by chrislovecnm**
- fixed line wrap
- fixed typos
- added updates for kops 1.7.1 release

### CODE REVIEW

File: docs/advisories/cve_2017_14491.md [hunk 1]
```diff
@@ -1,35 +1,59 @@
...
+A vulnerability in dnsmasq, used by kube-dns, requires an upgrade to the
+kube-dns component. This component is the default DNS component installed in
+Kubernetes.  The vulnerability may be externally exploitable. Links below exist
+with the full detail of the CVE. This exploit is not a Kubernetes specific vulnerability but exists in dnsmasq.
 
 ## Current kops Status
 
-As of 2017/10/08 `kops` fixes are not in a released `kops` version, but you are
-able to hotfix any Kubernetes cluster that is 1.4.x or higher.
+`kop` release 1.7.1 addresses this CVE.  This version of `kops` will upgrade and
```
Comment (justinsb) [inline docs/advisories/cve_2017_14491.md]: Typo: `kops`

File: docs/advisories/cve_2017_14491.md [hunk 2]
```diff
@@ -1,35 +1,59 @@
...
+create clusters. `kops` 1.8.x release does not contain the required changes.
```
Comment (justinsb) [inline docs/advisories/cve_2017_14491.md]: 1.8.0-alpha.1 doesn't, but alpha.2 and up will

File: docs/advisories/cve_2017_14491.md [hunk 3]
```diff
@@ -1,35 +1,59 @@
...
+
+## Upgrading Cluster
+
+To update a cluster.  The kube-dns deployment will be automatically upgraded.
```
Comment (justinsb) [inline docs/advisories/cve_2017_14491.md]: Dangling sentence

File: docs/advisories/cve_2017_14491.md [hunk 4]
```diff
@@ -1,35 +1,59 @@
...
+```
+
+Validate the change was applied to the deployment:
+
+```bash
+kubectl get deployment -n kube-system kube-dns \
+-o jsonpath='{.spec.template.spec.containers[?(@.name == "dnsmasq")].image}'
+```
+
+The upgrade is will occur once the channels utilty picks up the change within a
```
Comment (justinsb) [inline docs/advisories/cve_2017_14491.md]: grammar and typo in mintues

File: docs/advisories/cve_2017_14491.md [hunk 5]
```diff
@@ -126,4 +162,4 @@ _TODO_ if someone wants to provide the output.
 
 ## Thanks
 
-Thanks to @mikesplain, @chrislovecnm, @snoby, @justinsb, @3h4x
+Thanks to all that helped @mikesplain, @chrislovecnm, @snoby, @justinsb, @3h4x
```
Comment (justinsb) [inline docs/advisories/cve_2017_14491.md]: We should probably add aaronlevy here also

File: docs/advisories/cve_2017_14491.md [hunk 6]
```diff
@@ -1,35 +1,59 @@
...
-to fully tested version is recommend. We have had 1.4.x users upgrade successfully,
-but we cannot validate full production stability.  Local testing in a non-production
-environment is always recommended. We are not able to quatify the risk of using
-a non-tested version.
+Kubernetes versions 1.5.8, 1.6.11, 1.7.7, and 1.8.0 have been fully tested with
+the new version of `kube-dns` deployment.  Other versions should function, but
+upgrading to tested version is recommended. We have had 1.4.x users upgrade
+successfully, but we cannot validate full production stability.  Local testing
+in a non-production environment is always recommended. We are not able to
+quatify the risk of using a non-tested version.
```
Comment (justinsb) [inline docs/advisories/cve_2017_14491.md]: quantify

**Comment by chrislovecnm** [comment]
@justinsb PTAL, cleaned up english

File: docs/advisories/cve_2017_14491.md [hunk 7]
```diff
@@ -43,30 +70,37 @@ apply the hotfix.  kops should not downgrade the hotfix.
...
+### Kubernetes Versions 1.6.x and higher
 
 #### Installation of Hot Fix
 
 Apply the update to the container:
 
 ```bash
-kubectl set image deployment/kube-dns -n kube-system dnsmasq=gcr.io/google_containers/k8s-dns-dnsmasq-amd64:1.14.5
+kubectl set image deployment/kube-dns -n kube-system \
+ dnsmasq=gcr.io/google_containers/k8s-dns-dnsmasq-amd64:1.14.5
```
Comment (kooba) [inline docs/advisories/cve_2017_14491.md]: Shouldn't this be `k8s-dns-dnsmasq-nanny-amd64:1.14.5` ?

Comment (kooba) [reply docs/advisories/cve_2017_14491.md]: Nevermind, I think naming changed.

**Pull request merged**